packages/workspace/src/browser/workspace-frontend-contribution.ts

```diff
@@ -90,11 +90,16 @@ export class WorkspaceFrontendContribution implements CommandContribution, Keybi
         const updateWorkspaceStateKey = () => workspaceStateKey.set(this.updateWorkspaceStateKey());
         updateWorkspaceStateKey();
 
+        const workbenchStateKey = this.contextKeyService.createKey<WorkspaceState>('workbenchState', 'empty');
+        const updateWorkbenchStateKey = () => workbenchStateKey.set(this.updateWorkbenchStateKey());
+        updateWorkbenchStateKey();
+
         this.updateStyles();
         this.workspaceService.onWorkspaceChanged(() => {
             this.updateEncodingOverrides();
             updateWorkspaceFolderCountKey();
             updateWorkspaceStateKey();
+            updateWorkbenchStateKey();
```

Comment:
Do you expect that `workspaceState` and `workbenchState` may want to have different values in the future? In this code, they're updated in tandem and with the same values, so they could be combined.